ush/parsing_namelists_GOCART.sh

```diff
@@ -0,0 +1,52 @@
+#! /usr/bin/env bash
+
+# Disable variable not used warnings
+# shellcheck disable=SC2034
+GOCART_namelists() {
+  # copying GOCART configuration files
+  if [[  -n "${AERO_CONFIG_DIR}" ]]; then
+
+    local base_in
+    local fhout_aero_padded
+    fhout_aero_padded=$(printf "%02d" "${FHOUT_AERO}")
+    # Only instantaneous AOD is output right now
+    local inst_aod_freq="${fhout_aero_padded}0000"
```

Comment:
I think I am missing something.  Where does `inst_aod_freq` actually get written to `AERO_HISTORY.rc`?

Reply:
`atparse` does it on L34. This is how the other "parsing" scripts work too.